src/components/Dashboard/Dashboard.js

```diff
@@ -25,10 +38,10 @@ class Dashboard extends React.Component {
 
   filterBooks = () => {
     if (!this.state.currentFilterValue || !this.state.currentFilterType) {
-      this.setState({ filteredBooks: books });
+      this.setState({ filteredBooks: this.allBooks });
     } else {
       this.setState({
-        filteredBooks: books.filter(
+        filteredBooks: this.allBooks.filter(
```

Comment:
use the filters from the api. the query params for these are: `author` and `title`

Reply:
I'll do it in the redux I'll do it in the redux for books branch I'm about to pull request if you don't mind